### PR TITLE
Wishlist page

### PR DESCRIPTION
**Pull request by hinanshisuthar**
Wishlist management
- From the navbar, can navigate to my wishlist where all the products that one would like to buy are mentioned.
- On the product card,
-- Can remove the item from the wishlist
-- Add the item to the cart

### CODE REVIEW

File: src/pages/Wishlist Page/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
+@import url(../../styles/helper.css);
```
Comment (lannisterrr) [inline src/pages/Wishlist Page/wishlist.css:1]: you can import all your css in the a single file like app.css

File: src/pages/Wishlist Page/WishlistPage.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { useEffect } from "react";
+import { Navbar } from "../../components/Navbar"
+import { VertDismissCard } from "../../components/VertDismissCard";
+import { useWishlist } from "../../context/wishlist-context";
+import './wishlist.css'
+const WishlistPage = () => {
+    const {wishlistState} = useWishlist();
+return (
+    <>
+    <Navbar />
```
Comment (lannisterrr) [inline src/pages/Wishlist Page/WishlistPage.jsx:10]: Navbar is constant in all the pages , so it can be in app.js

**Review by lannisterrr — COMMENTED** [review]
Good file structure , use of reducer and context. Some suggestions made.

File: src/components/VertDismissCard.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+                <div className="prod-container">
+                    <img src={product.img} alt="product" />
+                    <span className="btn-badge-dismiss" onClick={() => wishlistDispatch({type: 'REMOVE_FROM_WISHLIST', payload: product})}><i className="fa-solid fa-heart px-sm" id="close-card" style={{color: '#f34e4e'}}></i></span> 
+                </div>
+                <div className="prod-content">
+                    <h5>{product.title}</h5>
+                    <div className="prod-info">
+                        <span className="prod-price">Rs {product.price}</span>
+                        <span className="ratings-con" style={{fontSize: '1rem'}}>
+                            <i className="fas fa-star"></i>{product.rating}
```
Comment (Prabhav-Sharma) [inline src/components/VertDismissCard.jsx:18]: Can destructure product properties once so you won't have to write product repeatedly

**Review by Prabhav-Sharma — APPROVED** [review]

**Pull request merged**